srcpkgs/python3-numpy-stl/template

```diff
@@ -1,25 +1,31 @@
 # Template file for 'python3-numpy-stl'
 pkgname=python3-numpy-stl
-version=2.12.0
+version=2.13.0
 revision=1
 wrksrc="numpy-stl-${version}"
 build_style=python3-module
 build_helper="numpy"
 hostmakedepends="python3-setuptools python3-Cython"
 makedepends="python3-devel"
+checkdepends="$depends python3-pytest-cov python3-docutils"
 depends="python3-setuptools python3-numpy python3-utils"
```

Comment:
This probably works only because templates are sourced multiple times. `checkdepends` should go after `depends` (both because of the logic and because of xlint).